src/Service_Mesh/handler.js

```diff
@@ -41,19 +34,28 @@ async function msgHandler(msg, success) {
                     organizationID: messageBody.whoDunIt.organizationID
                 },
             };
+            if (!config.email.host || !config.email.port || !config.email.email || !config.email.password) {
+                args.email = {
+                    to: messageBody.email.to,
+                    from: config.email.host,
+                    subject: messageBody.email.subject,
+                    body: messageBody.email.body,
+                    html: messageBody.email.html
+                }
+            }
             context.token = {
                 sub: args.gcID.toString()
             };
             try {
                 await createNotification(null, args, context, "{gcID, appID, actionLevel, email {to, from, subject, body, html}, online {titleEn, titleFr, descriptionEn, descriptionFr}, whoDunIt {gcID, teamID, organizationID}}");
```

Comment:
I'm concerned it may fail here as it's expecting email in the return info, but I'd like us to try this fix and see what happens.

Reply:
It should return null if no email data is in the notification